### DISCUSSION TITLE
3.8.16 logs exceptions that look like this

### DISCUSSION THREAD

**Discussion by kore-bots** [rabbitmq/rabbitmq-server#5723] [Other]
HI Team

We are seeing frequent crash with the above error. Could someone please help to find the root cause 

CRASH REPORT Process <0.20772.32> with 0 neighbours exited with reason: "stopping because dependent process <0.20770.32> died: shutdown" in gen_server:handle_common_reply/8 line 811

![image](https://user-images.githubusercontent.com/62376889/188641354-c3aa774a-f468-4b77-92b6-d75bf8f9990c.png)

**Comment by lukebakken** [comment]
Version 3.8.X is no longer supported, please upgrade.

In addition, it's nearly impossible for us to assist when given just a part of your log files.

**Comment by michaelklishin** [comment]
The process ins question is a direct Erlang client connection, so the "dependent process that died" must be one of: a Shovel, a Federation link or an MQTT, STOMP connection. All of those things can fail or be transient in nature because they represent client connections.

So anything that might cause a mass disconnect of MQTT clients, for example, a network interruption on the node, can produce exceptions like that in the log.

We cannot suggest much more from just that snippet.

**Reply by kore-bots** [reply]
Hi Michael

Thanks for the response. PFA the full logs for your review.

This has been started only in recent times & upgrading to latest version would be tedious process and that requires planning and other complexities. We have verified if this was caused due to a network interruption but there are other processes running on that node were working without any issues.

Can you suggest what could be the other possibilities that could cause this frequent crash?

I am also 
[29.txt](https://github.com/rabbitmq/rabbitmq-server/files/9528525/29.txt)
attaching full logs for your review. Can you help to review and let me know your thoughts.

**Reply by michaelklishin** [reply]
Other Erlang processes don't have to be affected by a network partition but things that back connections of various kinds, be it MQTT clients, STOMP clients, Shovel or Federation, will.
